shiny/express/ui/__init__.py

```diff
@@ -58,76 +54,71 @@
     input_action_link,
     input_checkbox,
     input_checkbox_group,
-    input_switch,
-    input_radio_buttons,
     input_dark_mode,
     input_date,
     input_date_range,
     input_file,
     input_numeric,
     input_password,
+    input_radio_buttons,
     input_select,
     input_selectize,
     input_slider,
-    bind_task_button,
+    input_switch,
     input_task_button,
     input_text,
     input_text_area,
-    panel_title,
     insert_accordion_panel,
+    insert_ui,
+    js_eval,
+    markdown,
+    modal,
+    modal_button,
+    modal_remove,
+    modal_show,
+    nav_spacer,
+    notification_remove,
+    notification_show,
+    panel_title,
     remove_accordion_panel,
+    remove_ui,
     update_accordion,
     update_accordion_panel,
-    update_sidebar,
     update_action_button,
     update_action_link,
     update_checkbox,
-    update_switch,
     update_checkbox_group,
-    update_radio_buttons,
```

Comment:
Seems you have a linter setting that is causing lots of changes here...can we avoid making a such large diff on this file?

Reply:
yeah, sorry. I tried hard to avoid format-on-save but missed this one

Reply:
@cpsievert It's fixed now